src/refract/TypeQueryVisitor.h

```diff
@@ -90,11 +90,11 @@ namespace refract
             }
 
             TypeQueryVisitor tq;
-            tq.visit(*e);
+            Visit(tq, *e);
 
             E type;
             TypeQueryVisitor eq;
-            type.content(eq);
+            Visitby(eq, type);
```

Comment:
There is something wrong it is `Visitby` literaly with small **b**. Probably not used template because there is nowhere `const IElement` used.
Will fix